### DISCUSSION TITLE
Wallboxen: Automatische Umschaltung der Phasenanzahl während des Ladens

### DISCUSSION THREAD

**Discussion by camueller** [camueller/SmartApplianceEnabler#282] [Allgemein]
Ich habe dazu mal einen [Issue erstellt](https://github.com/camueller/SmartApplianceEnabler/issues/281), wobei mir die Logik nicht klar ist, nach der sich die Umschaltpunkte definieren.

[Ursprünglich hat das "lars_bln" im PV-Forum gepostest.](https://www.photovoltaikforum.com/thread/104060-ger%C3%A4te-mit-home-manager-koppeln-via-semp-ethernet/?postID=2350197&highlight=phasen#post2350197)

Wenn der Sachverhalt geklärt ist, könnte ich das bei Gelegenheit mal in den SAE einbauen.

**Comment by RaphiBu** [comment]
Der Knackpunkt ist, dass es bei vielen Wallboxen ein Minimum der Amperes gibt (z.B. Warp 2: 6 A). D.h. wenn man mit drei Phasen lädt, ist das Minimum 4140 W. Ich vermute das hat seinen Ursprung bei den E-Autos, diese laden glaube ich auch erst ab 6 A.

D.h. wenn ich bei kleiner PV-Anlage nicht so viel Überschuss habe, muss ich auf eine Phase umsteigen. Dann kann ich wiederum den Bereich 1380 W bis 3680 W nutzen.

Dazwischen ist eine Lücke (3680 vs. 4140 W). Ich würde immer in dem unteren Bereich bleiben, bis genügend Power für den oberen zur Verfügung steht.

D.h. 1380..4139W -> 1 Phase, > 4140W -> 3 Phasen. Und die 6A-Grenze aus meinem Beispiel am besten konfigurierbar (oder ist die bei allen Wallboxen gleich?).

Ist das Käse oder eine gute Idee?

**Comment by camueller** [comment]
So sieht mein Plan für die Implementierung aktuell auch aus - siehe [Issue](https://github.com/camueller/SmartApplianceEnabler/issues/281),
